### PR TITLE
Create new job for checking PRs on che-dashboard project

### PR DESCRIPTION
**Pull request by musienko-maxim**
The che-dashboard project needs launching of happy-path test for verification PRs. This PR set up new job on PROW CI.

### CODE REVIEW

**Comment by musienko-maxim** [comment]
retest

File: ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-main__v7.yaml [hunk 1]
```diff
@@ -49,8 +49,8 @@ tests:
       cli: latest
       commands: |
         echo -e "[INFO] Test script content:"
-        cat .ci/oci-devworkspace-happy-path.sh
-        /bin/bash .ci/oci-devworkspace-happy-path.sh
+        cat .ci/oci-dashboard-happy-path.sh
+        /bin/bash .ci/ooci-dashboard-happy-path.sh
```
Comment (amisevsk) [inline ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-main__v7.yaml:53]: Is this rename intentional? It looks to be modifying the devworkspace happy path. Also, there's a typo on line 53 -- `ooci`.

**Review by amisevsk — COMMENTED** [review]
I've no issues with the PR but don't feel familiar enough with the details of configuring these jobs to approve/deny.

**Comment by musienko-maxim** [comment]
Duplicated:https://github.com/openshift/release/pull/20220
